src/geoserver/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""
+Functions and utilities for loading and serving data with geoserver.
+Imports here are accessible directly by `from src import geoserver`.
+"""
+from .database_layers import create_datastore_layer, create_db_store_if_not_exists
+from .geoserver_common import create_workspace_if_not_exists
+from .raster_layers import add_gtiff_to_geoserver, create_viridis_style_if_not_exists
+
+__all__ = [
+    "add_gtiff_to_geoserver",
+    "create_datastore_layer",
+    "create_db_store_if_not_exists",
+    "create_viridis_style_if_not_exists",
+    "create_workspace_if_not_exists",
+]
```

Comment:
I really like the use of this. I will definitely start incorporating this approach in my own code as well. Thanks for the inspiration!